server/api/auth/loginWithIdp.js

```diff
@@ -99,7 +99,9 @@ module.exports = (err, user, req, res, clientID, idpId) => {
         }
       }
     })
-    .catch(() => {
+    .catch(err => {
+      console.error(err);
```

Comment:
What kind of data the 'err' contains? Is there something that should not end up to logs?

Reply:
Maybe all the console.logs in the server routes could be checked and potentially enforce the usage of [log.js](https://github.com/sharetribe/ftw-daily/blob/master/server/log.js) instead?